httpx/utils.py

```diff
@@ -140,3 +142,31 @@ def parse_header_links(value: str) -> typing.List[typing.Dict[str, str]]:
             link[key.strip(replace_chars)] = value.strip(replace_chars)
         links.append(link)
     return links
+
+
+_LOGGER_INITIALIZED = False
+
+
+def get_logger(name: str) -> logging.Logger:
+    """Gets a `logging.Logger` instance and optionally
+    sets up debug logging if the user requests it via
+    the `HTTPX_DEBUG=1` environment variable.
+    """
+    global _LOGGER_INITIALIZED
+
+    # If the user wants debug logging on we set it up for them.
+    if not _LOGGER_INITIALIZED:
+        _LOGGER_INITIALIZED = True
+        if os.environ.get("HTTPX_DEBUG", "").lower() in ("1", "true"):
```

Comment:
Do we really need do validate what's in the environment variable, so long as it's non-empty? Just asking — I don't know if there are best practices about this.

```suggestion
        if os.environ.get("HTTPX_DEBUG"):
```

Reply:
Oh right, good point. :+1: